contrib.json

```diff
@@ -54,19 +54,44 @@
         [ "npm install", "Download dependencies"]
       ]
     },
+    "upstream": {
```

Comment:
We can. So, would `update local` just to the `pull` and `update remote` also do the `push`?